OpenRA.Mods.Common/Widgets/Logic/Editor/SaveMapLogic.cs

```diff
@@ -28,6 +28,20 @@ struct MapFileTypeInfo
 			public string UiLabel;
 		}
 
+		class SaveDirectory
+		{
+			public readonly Folder Folder;
+			public readonly string DisplayName;
+			public readonly MapClassification Classification;
+
+			public SaveDirectory(Folder folder, MapClassification classification)
+			{
+				Folder = folder;
+				DisplayName = Platform.UnresolvePath(Folder.Name);
```

Comment:
For completeness' sake:
```
[19:22:40]	<pchote> antares79: do you mean like System: mods/ra/maps? 
[19:50:55]	<antares79> pchote: sorry, was afk. I guess I'd leave out the path completely and have the drop down just say "System Maps" or "User Maps" 
[19:51:10]	<pchote> that won't work for mods that want multiple dirs
[19:51:19]	<pchote> e.g anything that extends one of our default mods :s
[19:52:11]	<antares79> pchote: but then they will have different labels, won't they? Or can you have multiple dirs for "User", for example?
[19:52:22]	<pchote> you can, yeah
[19:52:29]	<antares79> that sucks then
[19:53:02]	<antares79> pchote: then I got nothing. Probably best to leave it as it is, confusing as it may be
```